docs/ccoctl.md

```diff
@@ -92,7 +92,7 @@ $ oc adm release extract --credentials-requests --cloud=aws --to=./credrequests
 Then you can use `ccoctl` to process all CredentialsRequest objects in the `./credrequests` directory (from the example above)
 
 ```bash
-$ ccoctl aws create-all --name=<name> --region=<aws-region> --credentials-requests-dir=<path-to-directory-with-list-of-credentials-requests>
+$ ccoctl aws create-all --name=<name> --region=<aws-region> --credentials-requests-dir=<path-to-directory-with-list-of-credentials-requests> 
```

Comment:
Need to add optional parameter --create-private-s3-bucket here?

Reply:
fixed